Remove depencency on LSB. The redhat-lsb package pulls lot of dependencies
we do not care about (like X11, QT, ...). We want libcgroup to be able to
run without this bloat.

From: Jan Safranek <jsafrane@redhat.com>
diff -up libcgroup-0.36.2/scripts/init.d/cgconfig.in.lsb libcgroup-0.36.2/scripts/init.d/cgconfig.in
--- libcgroup-0.36.2/scripts/init.d/cgconfig.in.lsb	2010-03-11 13:05:16.000000000 +0100
+++ libcgroup-0.36.2/scripts/init.d/cgconfig.in	2010-06-22 14:33:43.000000000 +0200
@@ -44,7 +44,16 @@ servicename=cgconfig
 #
 # Source LSB routines
 #
-. /lib/lsb/init-functions
+. /etc/rc.d/init.d/functions
+log_success_msg () {
+    echo -n $*; success "$*"; echo
+}
+log_failure_msg () {
+    echo -n $*; failure "$*"; echo
+}
+log_warning_msg () {
+    echo -n $*; warning "$*"; echo
+}
 
 # read the config
 CREATE_DEFAULT=yes
diff -up libcgroup-0.36.2/scripts/init.d/cgred.in.lsb libcgroup-0.36.2/scripts/init.d/cgred.in
--- libcgroup-0.36.2/scripts/init.d/cgred.in.lsb	2010-01-17 00:27:36.000000000 +0100
+++ libcgroup-0.36.2/scripts/init.d/cgred.in	2010-06-22 14:33:43.000000000 +0200
@@ -40,7 +40,15 @@ CGRED_BIN=$sbindir/cgrulesengd
 
 # Source function library & LSB routines
 . /etc/rc.d/init.d/functions
-. /lib/lsb/init-functions
+log_success_msg () {
+    echo -n $*; success "$*"; echo
+}
+log_failure_msg () {
+    echo -n $*; failure "$*"; echo
+}
+log_warning_msg () {
+    echo -n $*; warning "$*"; echo
+}
 
 # Read in configuration options.
 if [ -f "/etc/sysconfig/cgred.conf" ] ; then
